include/seqan/basic/allocator_interface.h

```diff
@@ -204,6 +204,7 @@ allocate(T const &,
     if (data)
         SEQAN_PROADD(SEQAN_PROMEMORY, count * sizeof(TValue));
 #endif
+	memset((void*)data, 0, count * sizeof(TValue));
```

Comment:
https://travis-ci.org/seqan/seqan/builds/74372145
razers and micro_razers, but in GCC its all fine now. Very strange